Add 6.7.0 release highlights

danielmitterdorfer opened the pull request:
With this commit we document the highlight features of the Elasticsearch
6.7.0 release.

elasticmachine commented:
Pinging @elastic/es-docs

zuketo commented on docs/reference/release-notes/highlights-6.7.0.asciidoc:
```diff
@@ -0,0 +1,54 @@
+=== Cross-cluster replication
+
+We have introduced cross-cluster replication in 6.5.0 as a beta feature and it
+is now mature enough that we make it generally available. Cross-cluster
```
We introduced cross-cluster replication in 6.5.0 as a beta feature, it
is now mature enough to declare the feature GA (general availability).

zuketo commented on docs/reference/release-notes/highlights-6.7.0.asciidoc:
```diff
@@ -0,0 +1,54 @@
+=== Index lifecycle management
+
+We have introduced index lifecycle management in 6.6.0 as a beta feature and it
+is now mature enough that we make it generally available. The index lifecycle
```
Maybe the same wording from CCR: "We introduced index lifecycle management in 6.6.0 as a beta feature, it
is now mature enough to declare the feature GA (general availability).

zuketo commented on docs/reference/release-notes/highlights-6.7.0.asciidoc:
```diff
@@ -0,0 +1,54 @@
+=== {es-sql}
+
+We have introduced {es-sql} in 6.3.0 as an experimental feature and it is now
+mature enough that we make it generally available. This feature enables
```
We introduced {es-sql} in 6.3.0 as an experimental feature, it
is now mature enough to declare the feature GA (general availability).

danielmitterdorfer commented:
Thanks for the review. I've addressed all of your comments now in https://github.com/elastic/elasticsearch/pull/39826/commits/ee86b4f906c1d96ca67ac03561d8715558e91304.

danielmitterdorfer commented:
@elasticmachine run elasticsearch-ci/docs-check
